hackathonbaobab2021/core/experiment.py

```diff
@@ -59,33 +72,418 @@ def from_zipped_json(
     def solve(self, options: dict):
         raise NotImplementedError("complete this!")
 
-    def check_solution(self, list_tests: List[str] = None, **params) -> pt.SuperDict:
+    def check_solution(
```

Comment:
I might need some time to review the original checks in order to see if these checks perform the same operations to make sure.